glide.yaml

```diff
@@ -29,7 +32,7 @@ import:
   - pkg/types
   - pkg/conversion
 - package: k8s.io/kubernetes
-  version: v1.6.0
+  version: ebb8d6e0fadfc95f3d64ccecc36c8ed2ac9224ef # kube 1.7.0
```

Comment:
@jboyd01 just curious, do you know how ahead this SHA is from 1.7.0 release?  Or is it between 1.7.1 and 1.7.0 or in between 1.7.1 or 1.7.2 or even above 1.7.2?